templates/types/streaming/nextjs/app/components/ui/chat/chat-input.tsx

```diff
@@ -21,7 +21,11 @@ export default function ChatInput(
     | "messages"
     | "setInput"
     | "append"
-  >,
+  > & {
+    requestParams?: {
+      datasource?: string;
```

Comment:
we don't need to specify datasource here (we can send whatever) - also in the other files